### ISSUE TITLE
[Editor] [a11y] Color Picker does not read out colors with JAWS

### ISSUE THREAD

**Issue by tynan-dev** [telerik/kendo-angular#3148]
**Describe the bug**
When navigating through the Color Picker's options in the Editor while using the screen reader JAWS, the colors are not read out to the user. Blind or visually impaired users are unable to use this dropdown. 

**To Reproduce**

1. Open https://stackblitz.com/edit/angular-94gctb with JAWS screen reader running
2. Open the Color Picker dropdown
3. Using the keyboard, navigate through the color options.
4. Nothing is read out to the user. 

**Expected behavior**
JAWS should read out the color of the tile when the tile is active to allow blind or vision-impaired users to use this component.

**Screenshots**

**Browser**
 - OS: Windows 10
 - Browser: Chrome 86

**Additional context**
Adding an aria-label attribute with the appropriate color name to the tiles should resolve this a11y issue. The developer should also be allowed to provide a label to the color options array, or to let the label default to the color's hex code.

**Comment by Raisolution** [comment]
Issue is addressed in the latest version of the Inputs package.
